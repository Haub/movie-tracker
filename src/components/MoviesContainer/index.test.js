import React from 'react';
import { shallow } from 'enzyme';

import { MoviesContainer } from '.';

describe('MoviesContainer component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<MoviesContainer />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should match snapshot with props', () => {
    const movies = [{}];
    const history = {}
    const wrapper = shallow(<MoviesContainer movies={movies} history={history} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should display loading gif if no movies to display', () => {
    const movies = [];
    const history = {};
    const wrapper = shallow(<MoviesContainer movies={movies} history={history} />);
    expect(wrapper).toMatchSnapshot();
  });
});

