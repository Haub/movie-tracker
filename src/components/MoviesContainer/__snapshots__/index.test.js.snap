// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesContainer component should display loading gif if no movies to display 1`] = `
<Route>
  <Component />
</Route>
`;

exports[`MoviesContainer component should match snapshot 1`] = `
<Route>
  <Component />
</Route>
`;

exports[`MoviesContainer component should match snapshot with props 1`] = `
<Route>
  <Component />
</Route>
`;
